build-system/tasks/presubmit-checks.js

```diff
@@ -341,6 +341,7 @@ var forbiddenTerms = {
       'extensions/amp-access/0.1/amp-access.js',
       'extensions/amp-experiment/0.1/variant.js',
       'extensions/amp-user-notification/0.1/amp-user-notification.js',
+      'dist.3p/current/integration.js',
```

Comment:
@zhouyx When i run presubmit it shows that i'm using cid, so Hongfei suggested to whitelist it.